src/specklepy/objects/base.py

```diff
@@ -168,6 +168,7 @@ def __init_subclass__(
         initialization. This is reused to register each subclassing type into a class
         level dictionary.
         """
+        print(cls, super())
```

Comment:
Did you mean to leave this in?
Might be a bit too noisy

Reply:
i def did not want to leave that in :D